tests/sentry/replays/consumers/test_recording.py

```diff
@@ -22,7 +22,13 @@
 class RecordingTestCaseMixin:
     @staticmethod
     def processing_factory():
-        return ProcessReplayRecordingStrategyFactory()
+        return ProcessReplayRecordingStrategyFactory(
```

Comment:
should we also add a quick unit test that initializes the consumer with multi-processing just to sanity check?

Reply:
probably.  i'll see if it blows up :p